roles/nodejs-install/tasks/main.yml

```diff
@@ -0,0 +1,84 @@
+---
+
+  - name: nodejs | deps | install
+    apt: name={{ item }} state=present update_cache=yes
+    with_items:
+      - apt-utils
+      - build-essential
+      - autoconf
+      - libcppunit-dev
+      - libtool
+      - openssh-client
+      - openssh-server
+      - rsync
+      - sudo
+      - tar
+      - gzip
+      - wget
+    when: do_install|bool
+
+  - name: nodejs | install root | delete
+    file:
+        path: "{{ nodejs_install_root }}"
+        state: absent
+    when: remove_install_root|bool
+
+  - name: nodejs | install parent | create
+    file:
+        path: "{{ nodejs_install_parent }}"
+        state: directory
+    when: do_install|bool
+
+  - name: nodejs | source | dir | create
+    file:
+        path: "{{ nodejs_install_root }}/src"
+        state: directory
+    when: create_install_root|bool
+
+  - name: nodejs | repo | sync
+    command: >
+        rsync -avz --exclude=.git
+        "{{ nodejs_git_work_dir }}/"
+        "{{ nodejs_install_root }}/src"
+    when: create_install_root|bool
```

Comment:
What's with the rsync? Where do we actually git clone nodejs? And where is nodejs_git_work_dir coming from? I'm sure I'm missing something important :)